install/kubernetes/README.md

```diff
@@ -5,8 +5,9 @@ Please follow the installation instructions from [istio.io](https://istio.io/doc
 ## Directory structure
 This directory contains files needed for installing Istio on a Kubernetes cluster.
 
-* istio-rbac.yaml - apply this file in the begining to an RBAC enabled cluster
+* updateVersion.sh script can be run to generate installation files
```

Comment:
change "generate" to "regenerate" (and maybe say why would someone need to do that)